docker-compose.yml

```diff
@@ -1,6 +1,21 @@
 version: "2"
 
 services:
+  nginx:
```

Comment:
Could you elaborate how forwarding works here? I.e. how do requests to `test.example.com:NGINX_PORT` end up in `test.example.com:ZEEBE_PORT`.